packages/dbml-core/src/parse/mysql/parser.pegjs

```diff
@@ -84,10 +84,18 @@ TableSyntax
 		refs.push(ref);
 	});
 
-    // return statement
-    return indexes ? 
-    	{...table_name, fields, indexes} 
-      : {...table_name, fields}
+  let res = {...table_name, fields};
+
+	if (options && options.comment) {
+		// convert CREATE TABLE ... COMMENT '...' to Table notes
+		const note = options.comment;
+		res = { ...res, note }
```

Comment:
There's no need to copy res into itself here, it simply wastes memory
```suggestion
		res.note = note
```

Reply:
Refactored anh